common/persistence/sql/execution.go

```diff
@@ -197,18 +196,19 @@ func (m *sqlExecutionStore) createWorkflowExecutionTx(
 		return nil, serviceerror.NewInternal(fmt.Sprintf("CreteWorkflowExecution: unknown mode: %v", request.Mode))
 	}
 
-	if err := createOrUpdateCurrentExecution(ctx,
-		tx,
-		request.Mode,
-		request.ShardID,
-		namespaceID,
-		workflowID,
-		runID,
-		newWorkflow.ExecutionState.State,
-		newWorkflow.ExecutionState.Status,
-		newWorkflow.ExecutionState.CreateRequestId,
-		lastWriteVersion,
-	); err != nil {
+	row := sqlplugin.CurrentExecutionsRow{
+		ShardID:          shardID,
+		NamespaceID:      namespaceID,
+		WorkflowID:       workflowID,
+		RunID:            runID,
+		CreateRequestID:  newWorkflow.ExecutionState.CreateRequestId,
+		StartTime:        nil,
```

Comment:
Why aren't you setting the start time anywhere?

Reply:
I don't want to add logic to this PR.

Reply:
I see, next time, add a comment that this is coming later.